packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -72,19 +72,18 @@ import { SimpleDb, SimpleDbStore, SimpleDbTransaction } from './simple_db';
 const LOG_TAG = 'IndexedDbPersistence';
 
 /**
- * Oldest acceptable age in milliseconds for client metadata read from
- * IndexedDB. Client metadata and primary leases that are older than 5 seconds
- * are ignored.
+ * Oldest acceptable age in milliseconds for client metadata before the client
+ * is considered inactive and its associated data (such as the remote document
+ * cache changelog) is garbage collected.
  */
-const CLIENT_METADATA_MAX_AGE_MS = 5000;
+const MAX_CLIENT_AGE_MS = 30 * 60 * 1000; // 30 minutes
 
 /**
- * Oldest acceptable age in milliseconds for client metadata before it and its
- * associated data (such as the remote document cache changelog) can be
- * garbage collected. Clients that exceed this threshold will not be able to
- * replay Watch events that occurred before this threshold.
+ * Oldest acceptable metadata age for clients that may participate in the
+ * primary lease election. Clients that have not updated their client metadata
+ * within 5 seconds are not eligible to receive a primary lease.
  */
-const CLIENT_STATE_GARBAGE_COLLECTION_THRESHOLD_MS = 30 * 60 * 1000; // 30 Minutes
+const PRIMARY_ELECTION_MAX_CLIENT_AGE_MS = 5000;
```

Comment:
I'm still not wild about calling this an election...  Perhaps `CLIENT_ELIGIBLE_FOR_PRIMARY_MAX_AGE_MS` (or just `PRIMARY_ELIGIBLE_MAX_AGE_MS`)?